services/response_code.proto

```diff
@@ -1155,9 +1155,9 @@ enum ResponseCodeEnum {
   NEGATIVE_ALLOWANCE_AMOUNT = 290;
 
   /**
-   * The approveForAll flag cannot be set for a fungible token.
+   * [Deprecated] The approveForAll flag cannot be set for a fungible token.
    */
-  CANNOT_APPROVE_FOR_ALL_FUNGIBLE_COMMON = 291;
+  CANNOT_APPROVE_FOR_ALL_FUNGIBLE_COMMON = 291 [deprecated = true];
```

Comment:
what will we return if the token in NftAllowance with `approved_for_all = true`  is a fungible token? `FUNGIBLE_TOKEN_IN_NFT_ALLOWANCES`?

Reply:
That's correct @xin-hedera